Documentation/ApiOverview/FlexForms/Index.rst

```diff
@@ -27,6 +27,12 @@ over plugin features and what is to be rendered.
 Using FlexForms you have all the features of TCA, so it is possible
 to use input fields, select lists, show options conditionally and more.
 
+..  versionchanged:: 12.0
+    The superfluous array key `TCEforms` was removed and is not evaluated
+    anymore. Its sole purpose was to wrap real TCA definitions.
+
+    The tags `TCEforms` **should** be removed upon dropping TYPO3 v11 support.
```

Comment:
Maybe move this to the sentence above as it looks like not belong to the "versionchanged" part above in rendered doc.
We should also mention that in v12 a migration is in place, which will be removed in future TYPO3 versions.